terminusdb_client/woqlclient/woqlClient.py

```diff
@@ -11,6 +12,8 @@
 from .connectionConfig import ConnectionConfig
 from .dispatchRequest import DispatchRequest
 
+from ..woqlquery.woql_query import WOQLQuery
```

Comment:
looking at the Travis fail log, seems we cannot import WOQLQuery in WOQLClient as WOQLQuery imports WOQLClient, doing so will create an import loop and is not allow in Python.

Reply:
I have updated this with a local import of `WOQLClient` in `smart_query.py`. It allows both the tests and mypy to run without errors, which I think is a win. Let me know what you think of that solution.